### DISCUSSION TITLE
What is the best Kivy tag-line?

### DISCUSSION THREAD

**Discussion by Julian-O** [kivy/kivy-github-discussions#4] [General]
* According to the web-site:

   > Kivy: The Open Source Python App Development Framework.
   >
   > Build and distribute beautiful Python cross-platform GUI apps with ease.

* According to the documentation:

   > Kivy is an open source software library for the rapid development of applications equipped with novel user interfaces, such as multi-touch apps.

* According to the Readme: 

   > Kivy is an open source, cross-platform Python
framework for the development of applications that make use of innovative,
multi-touch user interfaces.
   > The aim is to allow for quick and easy interaction design and rapid prototyping
whilst making your code reusable and deployable.

* According to PyPI:

   > Kivy: A software library for rapid development of hardware-accelerated multitouch applications.

I propose to unify those to use more similar language.

I think the key messages to someone who has never heard of the project:
  * Open Source
  * Python
  * GUI
  * Cross-platform, including devices
  * Framework (not just a library)

I think the secondary messages are:
   * Multi-touch support
   * Beautiful
   * Easy
   * Rapid prototyping
   * Hardware-accelerated

The secondary messages don't all have to appear in the first sentence someone reads about Kivy.

So I propose:

> Kivy is an open-source Python framework for developing GUI apps that work cross-platform, including on Android, iOS and Raspberry Pi.

Any objections or suggestions?

**Comment by ElliotGarbus** [comment]
>Kivy is an open-source Python framework for developing GUI apps that work cross-platform, including on Android, iOS and Raspberry Pi.

I like the direction, but perhaps to make it more general...

Kivy is an open-source Python framework for developing GUI apps that work cross-platform, including desktop, mobile and embedded platforms.

**Reply by AndreMiras** [reply]
I also like the direction, but I would remove Raspberry Pi because it's too specific/reductive. However I do like that we mention the OS like Android and iOS specifically and we should mention Linux too.
I also like what misl6 suggested later here https://github.com/orgs/kivy/discussions/4#discussioncomment-7531563

**Comment by misl6** [comment]
Please consider that we need two separate sentences for the website. (`h1` and `h2`, for SEO purposes) + the page title (for the website).

So ATM is (and every line can be changed, separately if you have proposals):
**page title**: `Kivy: Cross-platform Python Framework for GUI apps Development`
**h1**: `Kivy: The Open Source Python App Development Framework.`
**h2**: `Build and distribute beautiful Python cross-platform GUI apps with ease.`
**additional paragraph, but with high relevance**: `Kivy runs on Android, iOS, Linux, macOS and Windows.`

Then we can have a separate tagline (a mashup of the previous ones) to use on docs and GitHub descriptions.

Important keywords for SEO to keep in mind (in a sorta order of importance - but I may have missed something):
- `Python`
- `build`, `develop`
- `apps`
- `cross-platform`
- `mobile`
- `desktop`
- `android`, `ios`, `windows`, `linux`, `macOS`
- `GUI`
- `framework`

**Reply by Julian-O** [reply]
Re: H1/H2 etc.

Agreed. I used the term "more similar language" rather than "make them all the same" because I realise we need to tailor it for different contexts.

I was really moving away from emphasizing "multi-touch" and "novelty" and "hardware acceleration" which will scare away others like me who were just looking for "You can run Python on both types of phone with one app."

**Reply by misl6** [reply]
> Agreed. I used the term "more similar language" rather than "make them all the same" because I realise we need to tailor it for different contexts.

I did not get that 😅

So ... ok, we're all in the same direction.

> I was really moving away from emphasizing "multi-touch" and "novelty" and "hardware acceleration" which will scare away others like me who were just looking for "You can run Python on both types of phone with one app."

Agree.

**Comment by FilipeMarch** [comment]
On [Kivy School](https://kivyschool.com/) we are using this sentence:

> Kivy School is the best and easiest way to learn Kivy. 
> Here you will learn how to build amazing apps for Android, iOS, macOS, Linux, and Windows with a single framework using Python! Code once and deploy everywhere!

Given that, I think that `Kivy: build amazing cross-platform apps for Android, iOS, macOS, Linux, and Windows using a single open-source Python framework.` is the most appealing shortest sentence.

**Comment by Julian-O** [comment]
Kivy was updated so the pypi, Readme and RST documents use consistent descriptions.

Similar work has happened for Buildozer, Plyer and PyJNIus. Kivy for iOS is coming soon.
